test/RestApi.test.ts

```diff
@@ -21,5 +27,32 @@ describe('REST API Test', () => {
             }
         });
     });
-}); 
+    
+    test('Validate CodeDeploy Deployment Config', () => {
```

Comment:
Can we make the title here more specific - "Validate CodeDeploy deploys in linear 10%/min configuration" or something to that extent